src/containers/Bookings.js

```diff
@@ -4,16 +4,33 @@ import Results from "../components/Results.js";
 import * as FakeBookings from "../data/fakeBookings.json";
 
 export default class Bookings extends Component {
-  search = () => {
+  constructor(props){
+    super(props);
+      this.state= {results:[]};
+
+
+  }
+  
+  searchId = () => {
+    this.setState({results:FakeBookings}) 
     console.info("to do!");
   };
+  searchName = () => {
+    this.setState({results:FakeBookings}) 
+    console.info("not to do!");
+  };
+  sortBy = (colum)=>{
+    FakeBookings.sort((a,b)=>{if(a[colum]>b[colum]){return 1}})
```

Comment:
@monaazami To debug why `number of days` is not filtering properly, put a console.log(colum) in the sortBy function and check what is the value of `colum` when you press `number of days`